Frontend/src/components/ContactUs/Faq.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Accordion from '../../components/accordion';
+import data from '../../utils/accordionData';
+
+
+const Faq = () => {
+  return (
+    <div className="justify-center items-center mb-5 bg-royalBlue pb-10">
+        <p className="text-white font-medium pb-10 pt-20 text-center">Frequently Asked Questions</p>
+        <div className="mb-5 mx-[30%]">
```

Comment:
The FAQ box breaks for smaller screen, please minimize the margin for smaller screen so that it can fit 
```suggestion
        <div className="mb-5 md:mx-[30%] mx-[10%]">
```

Reply:
Corrections and additions made.